documentation/api/query/v4/pql.markdown

```diff
@@ -0,0 +1,339 @@
+---
+title: "PuppetDB 3.2 » API » v4 » Puppet Query Language (PQL)"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/pql.html"
+---
+
+[entities]: ./entities.html
+[subquery]: #subqueries
+
+> **Experimental Feature**: This featureset is experimental, and it may be altered or removed in
+> future release.
+
```

Comment:
something at the top to specify how this language is going to be used would be helpful. Just reading this, I thought it might be a Puppet language feature.